zork++/src/lib/config_file/compiler.rs

```diff
@@ -88,6 +88,16 @@ pub enum CppCompiler {
     GCC, // Possible future interesting on support the Intel's C++ compiler?
 }
 
+impl CppCompiler {
```

Comment:
Perfect. Default module extensions are related with the compiler designed in the configuration file.